butterfree/migrations/migration.py

```diff
@@ -0,0 +1,62 @@
+"""Migration entity."""
```

Comment:
This file was not supposed to be in this file, right?

Reply:
Same thing here.